footer.php

```diff
@@ -1,4 +1,119 @@
-    <footer id="footer">
+    <footer id="footer" class="section">
+    <?php 
+      $logo_id = IGV_get_option('_igv_social_options', '_igv_metadata_logo_id');
+
+      $address = IGV_get_option('_igv_site_options', '_igv_office_address');
+
+      $email = IGV_get_option('_igv_site_options', '_igv_contact_email');
+      $phone = IGV_get_option('_igv_site_options', '_igv_contact_phone');
+
+      $mailchimp_url = IGV_get_option('_igv_site_options', '_igv_mailchimp_url');
+
+      $sponsors = IGV_get_option('_igv_sponsors_options', '_igv_sponsors_group');
+
+      $facebook = IGV_get_option('_igv_social_options', '_igv_socialmedia_facebook_url');
+      $twitter = IGV_get_option('_igv_social_options', '_igv_socialmedia_twitter');
+      $instagram = IGV_get_option('_igv_social_options', '_igv_socialmedia_instagram');
+    ?>
+      <div class="container">
+
+        <div class="row">
+
+          <div class="col col-l col-l-1 align-center">
+        <?php if (!empty($logo_id)) { ?>
+            <a href="<?php echo home_url(); ?>"><?php echo wp_get_attachment_image($logo_id); ?></a>
```

Comment:
`wp_get_attachment_image` gets wp default size 'thumbnail' by default. that fits the container ok